src/index.ts

```diff
@@ -30,20 +34,23 @@ export const createJsonSchemaTransform = ({ skipList }: { skipList: readonly str
     }
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    if (params) transformed.params = zodToJsonSchema(params as any);
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    if (body) transformed.body = zodToJsonSchema(body as any);
+    if (params) transformed.params = zodToJsonSchema(params as any, zodToJsonSchemaOptions);
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    if (querystring) transformed.querystring = zodToJsonSchema(querystring as any);
+    if (body) transformed.body = zodToJsonSchema(body as any, zodToJsonSchemaOptions);
+    if (querystring)
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      transformed.querystring = zodToJsonSchema(querystring as any, zodToJsonSchemaOptions);
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    if (headers) transformed.headers = zodToJsonSchema(headers as any);
+    if (headers) transformed.headers = zodToJsonSchema(headers as any, zodToJsonSchemaOptions);
 
     if (response) {
       transformed.response = {};
+
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       for (const prop in response as any) {
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        transformed.response[prop] = zodToJsonSchema((response as any)[prop]);
+        const transformedResponse = zodToJsonSchema(response[prop] as any, zodToJsonSchemaOptions);
```

Comment:
❓

Looks like the pipeline found something wrong with this line? But I can't see it... probably worth investigating locally.

Reply:
O it probably wants:

```suggestion
        const transformedResponse = zodToJsonSchema((response as any)[prop], zodToJsonSchemaOptions);
```

I think because `response` is `unknown` so you can't index it by anything.

Reply:
thanks, that was it